Helmsman/dsf/mosip-platform-java21/external-dsf.yaml

```
@@ -337,12 +346,16 @@ apps:
       persistence.size: "8Gi"
       zookeeper.enabled: "true"
       zookeeper.replicaCount: "5"
+      zookeeper.resources.requests.cpu: "150m"
+      zookeeper.resources.requests.memory: "600Mi"
+      zookeeper.resources.limits.cpu: "300m"
+      zookeeper.resources.limits.memory: "1200Mi"
+      zookeeper.replicaCount: "5"
```

YAMLlint (1.38.0)

[error] duplication of key "zookeeper.replicaCount" in mapping

(key-duplicates)